File: BackEnd/eMenka/eMenka.Domain/Classes/Driver.cs
using System;

namespace eMenka.Domain.Classes
{
    public class Driver
    {
        public int Id { get; set; }
        public Person Person { get; set; }
        public int PersonId { get; set; }
        public FuelCard FuelCard { get; set; }
        public int? FuelCardId { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime? EndDate { get; set; }
    }
}

File: eFenkaFrontend/src/app/models/driver/driver.ts
import {FuelCard} from './../fuel-card/fuel-card';
import {Person} from '../person/person';

export class Driver {
  id: number;
  person: Person;
  fuelCard: FuelCard;
  startDate: Date;
  endDate: Date;

  constructor(id: number, person: Person, fuelCard: FuelCard, startDate: Date, endDate: Date) {
    this.id = id;
    this.person = person;
    this.fuelCard = fuelCard;
    this.startDate = startDate;
    this.endDate = endDate;
  }
}
